server/controllers/RunController.js

```diff
@@ -0,0 +1,41 @@
+const RunService = require('../services/RunService');
+
+async function configureRuns(req, res) {
+    const cycle = req.body.data;
+    try {
+        const savedRun = await RunService.configureRuns(cycle);
```

Comment:
perhaps this is "activeRuns"

Reply:
or maybe what we will return is an object that explains the changes that will be made with the new configuration.